### PR TITLE
[Bugfix] #7386 Make Google Maps component adaptive to parent div dimensions

### PR DESCRIPTION
**Pull request by jaworovcki**
**Check Issue** [#7386](https://github.com/ita-social-projects/GreenCity/issues/7386)

## Pull Request details:
- Check the mock-up of the Places page - the google map component view differs from the current one on the client
- On the .map-container inner div of the <google-map> the height and widht were hardcoded to 500px
- Query the class and give the width:100% and height:100% properties to behaves as adaptive container

### CODE REVIEW

File: src/app/main/component/places/places.component.ts [hunk 1]
```diff
@@ -332,6 +338,14 @@ export class PlacesComponent implements OnInit, OnDestroy {
       });
   }
 
+  modifyMapContainer() {
```
Comment (dmytro-kryvyi) [inline src/app/main/component/places/places.component.ts]: Add a return type (:void) if this method is really necessary and you can't specify this styles in the scss file instead

Comment (jaworovcki) [reply src/app/main/component/places/places.component.ts]: add return type (the styles cannot be added to scss file couse the selector generates after the page was rendered)

Comment (jaworovcki) [reply src/app/main/component/places/places.component.ts]: so the only ways are to make ViewEncupsulation.None(which is a bad idea) or to use the DOM to add the styles in afterViewInit

Comment (dmytro-kryvyi) [reply src/app/main/component/places/places.component.ts]: You can achieve the same result with built-in features. Just use the width and height properties of google-map component. 
image


Also you can use
 `:host ::ng-deep .class {}` 
to rewrite style and save encapsulation but there is no need for this in this case.

**Comment by jaworovcki** [comment]
Update the code with all the changes requested

**Review by dimoybiyca — APPROVED** [review]

**Review by Tadei-Mykola — APPROVED** [review]

**Review by DenysLeontiev — APPROVED** [review]
Nice

**Review by hnativlyubomyr — APPROVED** [review]

**Pull request merged**